runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler.cpp

```diff
@@ -111,10 +111,15 @@ OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler_initCifNativeTh
 		rc = GOTO_THROW_CURRENT_EXCEPTION;
 		setCurrentException(currentThread, J9VMCONSTANTPOOL_JAVALANGINTERNALERROR, NULL);
 		goto done;
-	/* Only intended for strut as the primitive's ffi_type is non-null */
+	/* Only intended for struct as the primitive's ffi_type is non-null */
```

Comment:
Missing period.

Reply:
Resolved in https://github.com/eclipse-openj9/openj9/compare/6322f343e9e88af4b64f7d08586a47a7ecb89b32..583867825be4477a8319f21ffb952780631fbeb3